wordpress/wordpress-api.ts

```diff
@@ -1,10 +1,23 @@
 
 /**
- * `GET & POST PAGES /wp/v2/pages`
+ * `GET PAGES /wp/v2/pages`
  */
 export const WORD_PRESS_API_GET_PAGE_PATH = '/wp-json/wp/v2/pages';
```

Comment:
It would be better to have GET and POST routes named separately. Otherwise it would be hard to later change.